Lucidtech/Las/Client.cs

```diff
@@ -746,17 +750,22 @@ public object CreateBatch(string? name = null, string? description = null)
         /// <returns>Transition executions response from the REST API</returns>
         public object ListTransitionExecutions(
             string transitionId,
-            List<string>? status = null,
+            List<string>? statuses = null,
```

Comment:
Wouldn't it be less code to maintain if the previous method calls this method with its status as a list with one element?